install.ps1

```diff
@@ -493,117 +526,36 @@ if (-not $noGui.IsPresent) {
-            $selectedPackagesBox.EndUpdate()
-        }
-        $numSelectedLabel.text = "Total: $($selectedPackagesBox.Items.count)"
-
-        $unselectedPackagesBox.BeginUpdate()
-        $unselectedPackagesBox.Items.Clear()
-        $unselectedPackagesBox.EndUpdate()
-        $numUnselectedLabel.text = "Total: $($unselectedPackagesBox.Items.count)"
-    }
-
-    function Remove-SelectedPackages {
-        $selectedPackages = $selectedPackagesBox.SelectedItems
-        foreach($package in $selectedPackages)
-        {
-            $unselectedPackagesBox.BeginUpdate()
-            $unselectedPackagesBox.Items.Add($package) | Out-Null
-            $unselectedPackagesBox.EndUpdate()
-        }
-        $numUnselectedLabel.text = "Total: $($unselectedPackagesBox.Items.count)"
-
-        $selectedPackagesBox.BeginUpdate()
-        while ($selectedPackagesBox.SelectedItems.count -gt 0) {
-            $selectedPackagesBox.Items.RemoveAt($selectedPackagesBox.SelectedIndex)
-        }
-        $selectedPackagesBox.EndUpdate()
-        $numSelectedLabel.text = "Total: $($selectedPackagesBox.Items.count)"
-    }
-
-    function Remove-AllPackages {
-        foreach($package in $selectedPackagesBox.Items)
-        {
-            $unselectedPackagesBox.BeginUpdate()
-            $unselectedPackagesBox.Items.Add($package) | Out-Null
-            $unselectedPackagesBox.EndUpdate()
-        }
-        $numUnselectedLabel.text = "Total: $($unselectedPackagesBox.Items.count)"
-
-        $selectedPackagesBox.BeginUpdate()
-        $selectedPackagesBox.Items.Clear()
-        $selectedPackagesBox.EndUpdate()
-        $numSelectedLabel.text = "Total: $($selectedPackagesBox.Items.count)"
-    }
-
+    
     # Gather lists of packages (i.e., available, already installed, to install)
     $excludedPackages = @("flarevm.installer.vm", "common.vm")
     $installedPackages = (Get-InstalledPackages).Name
     $packagesToInstall = $configXml.config.packages.package.name | Where-Object { $installedPackages -notcontains $_ }
-    $allPackages = (Get-InstallablePackages).Name | Where-Object { $packagesToInstall -notcontains $_ -and $installedPackages -notcontains $_ -and $excludedPackages -notcontains $_}
+    #$allPackages = (Get-InstallablePackages).Name | Where-Object { $packagesToInstall -notcontains $_ -and $installedPackages -notcontains $_ -and $excludedPackages -notcontains $_}
     $envs = [ordered]@{}
     $configXml.config.envs.env.ForEach({ $envs[$_.name] = $_.value })
 
-    $form                   = New-Object system.Windows.Forms.Form
-    $form.ClientSize        = New-Object System.Drawing.Point(717,740)
-    $form.text              = "FLARE VM Install Customization"
-    $form.TopMost           = $true
-    $form.MaximizeBox       = $false
-    $form.FormBorderStyle   = 'FixedDialog'
-    $form.StartPosition     = 'CenterScreen'
+    
+    #needed to use Find-Package
+    install-packageprovider nuget -force
```

Comment:
Make the installation of nuget consistency (same code structure) with the installation of other dependencies like boxstarter and chocolatey. Concretely: check if it is installed if possible, and print a message that is being installed and installed successfully instead of the output:
```
Name                           Version          Source           Summary
----                           -------          ------           -------
nuget                          2.8.5.208        https://onege... NuGet provider for the OneGet meta-package manager
```